src/is/L42/flyweight/C.java

```diff
@@ -0,0 +1,112 @@
+// Generated by delombok at Sun Jun 13 15:12:41 NZST 2021
+package is.L42.flyweight;
+
+import java.io.ObjectStreamException;
+import java.util.HashMap;
+import java.util.Map;
+
+import is.L42.common.Constants;
+import is.L42.generated.LDom;
+import is.L42.perftests.PerfCounters;
+import is.L42.visitors.CloneVisitor;
+import is.L42.visitors.CollectorVisitor;
+import is.L42.visitors.Visitable;
+
+public final class C implements LDom, Visitable<C> {
+  private static final long serialVersionUID = 1477016806296313145L;
+
+  @Override
+  public C accept(CloneVisitor cv) {
+    return cv.visitC(this);
+  }
+
+  @Override
+  public void accept(CollectorVisitor cv) {
+    cv.visitC(this);
+  }
+
+  @Override
+  public String toString() {
+    return Constants.toS.apply(this);
+  }
+
+  @Override
+  public boolean wf() {
+    return Constants.wf.test(this);
+  }
+
+  private final String inner;
+  private final int uniqueNum;
+  
+  private static record CI(String inner, int uniqueNum) {}
+  private static Map<CI, C> created = new HashMap<>();
+  public static C newC(final String inner, final int uniqueNum) {
+    CI ci = new CI(inner, uniqueNum);
+    if(PerfCounters.isEnabled()) {
+      PerfCounters.inc("invoke.C.init.total");
+      if(!created.containsKey(ci)) {
+        PerfCounters.inc("invoke.C.init.total.unique");
+        }
+      }
+    return created.computeIfAbsent(ci, ci2->new C(ci2.inner, ci2.uniqueNum));
+    }
+  Object readResolve() throws ObjectStreamException {
+    return newC(this.inner, this.uniqueNum);
+    }
+  
+  @java.lang.SuppressWarnings("all")
+  private C(final String inner, final int uniqueNum) {
+    this.inner = inner;
+    this.uniqueNum = uniqueNum;
+  }
+
+  @java.lang.SuppressWarnings("all")

```

Comment:
Ok, "all" is a leftover of Lombock... still, remove all the un needed suppress warnings and put the methods in a reasonable order